scripts/simplifiedUI/ui/simplifiedUI.js

```diff
@@ -457,9 +457,16 @@ function onGeometryChanged(rect) {
     }
 }
 
-function ensureFirstPersonCameraInHMD(isHMDMode) {
+var TIMEOUT_BEFORE_REHIDE_TOOLBAR_MS = 700
+function onDisplayModeChanged(isHMDMode) {
     if (isHMDMode) {
         Camera.setModeString("first person");
+    } else {
```

Comment:
```suggestion
    } else if (Settings.getValue("simplifiedUI/keepExistingUIAndScripts", false)) {
```

Sorry, I forgot about this yesterday!

Reply:
Ack

Reply:
Ack